src/Common/CmfPortalSession.cs

```diff
@@ -13,25 +13,27 @@ public abstract class CmfPortalSession : ISession
         private const string _loginTokenFileName = "cmfportaltoken";
         private static readonly string _loginCredentialsDir = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData), _cmfPortalDirName);
         private static readonly string _loginCredentialsFilePath = Path.Combine(_loginCredentialsDir, _loginTokenFileName);
-        private string accessToken = null;
+        private string token = null;
 
         public LogLevel LogLevel { get; protected set; } = LogLevel.Information;
 
         public IConfiguration Configuration { get; set; }
+       public string AccessToken { get; set; }
 
-        protected string AccessToken
+        //depending on the login strategy, this token can be the PAT or the Refresh Token
```

Comment:
This comment should be added in the form of a c# comment or else it won't show up in the intelisense. Also, since we're using an interface, it should be moved to that interface or else it won't show up as well